### DISCUSSION TITLE
Welcome to 30-Days-of-Code Discussions!

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [Codezilla-Club/30-Days-of-Code#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Share ideas
  * Engage with other community members
  * Ask questions
  * Welcome others and are open-minded
  
Remember that this is a community we build together 💪

To get started, comment below with an introduction of yourself and tell us about what you do with this community!

**Comment by srinivas1904code** [comment]
hi I am srinivas, I am having a doubt in weeek one assignment , i.e.., should we consider different cases for calculating area(like right angle triangle,equilateral,etc.,) or just 3,4 sided regular polygon??
